### DISCUSSION TITLE
Why not use System.Text.Json?

### DISCUSSION THREAD

**Discussion by AKANexus** [lawrencea13/LethalCompanySaveManager#10] [Ideas]
Quick question: Rather than use string iterators to work with the decrypted save file, have you tried deserializing the decrypted json with `System.Text.Json.Serializer`, or does the save data have a peculiarity that prevents that approach?

**Comment by lawrencea13** [comment]
When I originally wrote this I was fairly green. I had some guidance and was advised that it uses a proprietary format. After some experience it looks like it doesn’t. It’s a little odd but json should work. With that said, I don’t see a need to rewrite it since there aren’t plans for more savegame changes unless the game adds more stuff to saves.

with that said, it’s open source and if someone would like to change it over to json with a pull request, I’d happily merge that into main.
